admin_pages/registration_form/help_tabs/registration_form_questions_overview_table_column_headings.help_tab.php

```diff
@@ -1,59 +1,57 @@
-<p><strong><?php _e('Questions Overview Table Column Headings', 'event_espresso'); ?></strong></p>
-<p>
+<p><strong><?php esc_html_e('Questions Overview Table Column Headings', 'event_espresso'); ?></strong></p>
 <ul>
 <li>
-<strong><?php _e('ID', 'event_espresso'); ?></strong><br />
-<?php _e('This is the numerical ID for the question. This value is used internally for Event Espresso.', 'event_espresso'); ?>
+<strong><?php esc_html_e('ID', 'event_espresso'); ?></strong><br />
+<?php esc_html_e('This is the numerical ID for the question. This value is used internally for Event Espresso.', 'event_espresso'); ?>
```

Comment:
Should we use the new brand name helper for the `Event Espresso` brand string here and elsewhere in this set of changes?